authors/apps/articles/models.py

```diff
@@ -27,4 +31,17 @@ def save(self, *args, **kwargs):
         super(Article, self).save(*args, **kwargs)
 
     def __str__(self):
-        return self.title
\ No newline at end of file
+        return self.title
+
+    def average_rating(self):
+        from authors.apps.ratings.models import Rating
```

Comment:
Hello @jennizalwango , please consider putting this import at the top of the file

Reply:
oh @Emmanuelaaron, l had to put that import there because l was getting errors and the solution l had was to use circular Imports and that is what you are seeing @ that line, You could read more about them [https://stackabuse.com/python-circular-imports/](url)